### DISCUSSION TITLE
HTTP Endpoint with parameters (designer)

### DISCUSSION THREAD

**Discussion by Aoshua** [elsa-workflows/elsa-core#1011] [Q&A] [answered]
Using the designer, is there a way to make the starting HTTP Endpoint activity take a path including parameters? Something like `/workflowA?id=`

**Comment by sfmskywalker** [comment, marked as answer]
Yes, you can send querystring parameters to the workflow without needing to change the path in the HTTP Endpoint activity.

You can then read the querystring parameters from any other activity, because it will be part of HTTP Endpoint’s output.

Route segments on the other hand are not supported yet, but I do intend to make that work too with ASP.NET Core’s routing API.

**Comment by atorres16** [comment]
From https://elsa-workflows.github.io/elsa-core/docs/expressions/expressions-javascript#querystring

You just call the integrated function:

```javascript
queryString("nameOfParameter");
```
